### ISSUE TITLE
Do not connect bot

### ISSUE THREAD

**Issue by XOTTABY4** [FilipeBorges1993/Booh-Botnet#1]
After installation and compilation, the botnet does not appear in the panel. 
I tried to install the panel on both the local and remote server, to result in the same.

**Comment by FilipeBorges1993** [comment]
Humm, did you set the server ip as base64 crypted ?

**Comment by XOTTABY4** [comment]
Yes of course, and in the sniffer the request goes to the right address

**Comment by FilipeBorges1993** [comment]
Humm its on the crypth controller. It´s producing an extra "/" caracter.  On the bot !try to produce an other acount on the painel. (fromat the db and create an new panel acount)

**Comment by XOTTABY4** [comment]
I created a new account in the panel but I still do not have any input, can you help ??
